dynamic-plugins/wrappers/backstage-plugin-techdocs/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "backstage-plugin-techdocs",
+  "version": "1.8.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin"
+  },
+  "scripts": {
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test --passWithNoTests --coverage",
+    "clean": "backstage-cli package clean",
+    "export-dynamic": "janus-cli package export-dynamic-plugin"
+  },
+  "dependencies": {
+    "@backstage/core-plugin-api": "1.7.0",
+    "@backstage/plugin-catalog-react": "1.8.5",
+    "@backstage/plugin-search-react": "1.7.1",
+    "@backstage/plugin-techdocs": "1.8.0",
+    "@backstage/plugin-techdocs-module-addons-contrib": "1.1.1",
+    "@backstage/plugin-techdocs-react": "1.1.12"
+  },
+  "peerDependencies": {
+    "react": "^16.13.1 || ^17.0.0"
+  },
+  "devDependencies": {
+    "@backstage/cli": "0.23.1",
+    "@janus-idp/cli": "1.4.2"
+  },
+  "files": [
+    "dist",
+    "dist-scalprum"
+  ],
+  "scalprum": {
+    "name": "backstage.plugin-techdocs",
+    "exposedModules": {
+      "PluginRoot": "./src/index.ts"
```

Comment:
We default to a whole scalprum object. Not to its partials. Therefore if we omit the `scalprum` key in the `package.json` we would end up with:

```json
"scalprum": {
  "name": "backstage-plugin-techdocs",
  "exposedModules": {
    "PluginRoot": "./src/index.ts"
  }
}
```

Instead of:

```json
"scalprum": {
  "name": "backstage.plugin-techdocs",
  "exposedModules": {
    "PluginRoot": "./src/index.ts"
  }
}
```

Notice the asset name:

```diff
-   "name": "backstage-plugin-techdocs",
+   "name": "backstage.plugin-techdocs",
```

We want to be consistent with the naming convention and we're writing an upstream package wrapper, our wrapper NPM package name is not `@backstage/plugin-techdocs` but `backstage-plugin-techdocs`. The default name inferred by `janus-cli` would be different and we don't want that. So we have to write the config manually here. As we do for all the other wrapped upstream plugins.

https://github.com/janus-idp/backstage-plugins/blob/a80aed93dc905b939f989a8ea5ced8ac49eecba5/packages/cli/src/commands/export-dynamic-plugin/frontend.ts#L36